src/modules/shared/const/i18n.tsx

```diff
@@ -0,0 +1 @@
+export const DEFAULT_LOCALE = 'nl';
```

Comment:
Ik ga meestal voor `nl_BE` & `fr_BE`, [language-code](https://nl.wikipedia.org/wiki/Lijst_van_ISO_639-codes)_[country-code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)
https://www.ibm.com/docs/en/radfws/9.7?topic=overview-locales-code-pages-supported

Reply:
Dit is een verwijzing naar wat er gebruikt wordt in de `next.config.js` je kan daar meerdere locales laten wijzen naar 1 url: https://nextjs.org/docs/advanced-features/i18n-routing#domain-routing 
Dus dit kunnen we in de toekomst nog altijd aanpassen eens we meertaligheid ondersteunen.